Libplanet/Net/Messages/BlockHashes.cs

```diff
@@ -5,40 +5,40 @@
 
 namespace Libplanet.Net.Messages
 {
-    internal class GetData : Message
+    internal class BlockHashes : Message
     {
-        public GetData(IEnumerable<HashDigest<SHA256>> hashes)
+        public BlockHashes(IEnumerable<HashDigest<SHA256>> hashes)
         {
-            BlockHashes = hashes;
+            Hashes = hashes;
         }
 
-        public IEnumerable<HashDigest<SHA256>> BlockHashes { get; }
+        public IEnumerable<HashDigest<SHA256>> Hashes { get; }
 
-        protected override MessageType Type => MessageType.GetData;
+        protected override MessageType Type => MessageType.BlockHashes;
 
         protected override IEnumerable<NetMQFrame> DataFrames
         {
             get
             {
                 yield return new NetMQFrame(
-                    NetworkOrderBitsConverter.GetBytes(BlockHashes.Count()));
+                    NetworkOrderBitsConverter.GetBytes(Hashes.Count()));
 
-                foreach (HashDigest<SHA256> hash in BlockHashes)
+                foreach (var hash in Hashes)
                 {
                     yield return new NetMQFrame(hash.ToByteArray());
                 }
             }
         }
 
-        internal static Message Parse(NetMQFrame[] frames)
+        public static Message Parse(NetMQFrame[] frames)
```

Comment:
Is there any reason to change to public differently from other Message classes?

Reply:
Nope, I'll change equally.